README.md

```diff
@@ -10,5 +48,10 @@ the function and run automatically when you commit through a GitHub action.
 
 ## Runtime Analysis
 
+Prefice: I put a significant time into attempting to utilize a parrel reduce to implement mergesort, however it was incredebly difficult to find any information on that subject, in addition to most topics being focused on hadoops mapReduce sort. This has, however, offered me a lot of insight into how that particular sort works and mergesort as well as the they are both divide and conquer. Consequently I also learned that there is a relation to map and reduce, if the sorting is reduce sided it is mergesort, if it is map sided it is closer to quicksort.
+
 What is the span of the parallel program, in terms of worst-case $\Theta$? Hint:
 It may help to consider the DAG of the parallel program.
+
+Given that mergeSort() recursively splits itself (creating a new "process/thread" each time it does so until the arrays are less that one) the longest path would be logn so then the span is $\Theta(logn)$. 
```

Comment:
But are you doing constant work in each node on that path?

Reply:
Ahhh, merge is variable, let me correct that

Reply:
I had the correct answer, just not in the right place.